### ISSUE TITLE
Learning issue

### ISSUE THREAD

**Issue by prachikashikar** [CEA-LIST/N2D2#51]
Hello how can I remove the following error?

prachi@prachi-V530-15ICB:~/N2D2$ ./build/bin/n2d2 ./models/mnist24_16c4s2_24c5s2_150_10.ini -learn 600000 -log 10000
Option -log: number of steps between logs [10000]
Option -learn: number of backprop learning steps [600000]
Loading network configuration file ./models/mnist24_16c4s2_24c5s2_150_10.ini
Warning: to use Frame_CUDA models, N2D2 must be compiled with CUDA enabled.
*** Using Frame model instead. ***
Notice: Unused section sp in INI file
Notice: Unused section sp.Transformation in INI file
Notice: Unused section conv1 in INI file
Notice: Unused section conv2 in INI file
Notice: Unused section fc1 in INI file
Notice: Unused section fc1.drop in INI file
Notice: Unused section fc2 in INI file
Notice: Unused section softmax in INI file
Notice: Unused section softmax.Target in INI file
Notice: Unused section common.config in INI file
Time elapsed: 0.000383066 s
Error: Could not open images file: /local/prachi/n2d2_data/mnist/train-images-idx3-ubyte

**Comment by prachikashikar** [comment]
It looks like solved in #24 . But I dont get how it is done. Do reply.

**Comment by Djip007** [comment]
try:
```
ls -l /local/prachi/n2d2_data/mnist/train-images-idx3-ubyte
```
Not sure In your screenshot on #54 I see something like:
```
~/N2D2_DATA/mnist/train-images-idx3-ubyte
``` 
You may have wrong export N2D2_DATA env var before try the training...
But pay be wrong...

**Comment by prachikashikar** [comment]
Thank you so much @Djip007 . This worked.
